MonMaraicher/Features/Map/MapView.swift

```diff
@@ -54,6 +54,8 @@ extension MapView {
                 .background(Circle()
                     .fill(.thinMaterial))
         }
+        .alert(viewModel.titleError, isPresented: $viewModel.isErrorSearchFarmerPresented, actions: {
```

Comment:
Why leave action empty?

Reply:
I could have put a button to manage each alert message.

Change the distance if no farmer found and go to the settings if the user did not accept the location but it took a little more time to create these methods.

I’m going to create them soon

Reply:
Add a todo then, easier for me to understand ;)